wsmaster/che-core-api-factory-github/src/main/java/org/eclipse/che/api/factory/server/github/GithubFactoryParametersResolver.java

```diff
@@ -36,55 +35,42 @@
 import org.eclipse.che.api.workspace.server.devfile.URLFetcher;
 import org.eclipse.che.api.workspace.shared.dto.ProjectConfigDto;
 import org.eclipse.che.api.workspace.shared.dto.devfile.ProjectDto;
-import org.eclipse.che.commons.annotation.Nullable;
 
 /**
  * Provides Factory Parameters resolver for github repositories.
  *
  * @author Florent Benoit
  */
 @Singleton
-public class GithubFactoryParametersResolver extends RawDevfileUrlFactoryParameterResolver {
+public class GithubFactoryParametersResolver implements FactoryParametersResolver {
```

Comment:
Why do we need this changes?

Reply:
To not to override the `priority()` function for each SCM resolver. This function is default in the interface and returns `DEFAULT` priority for each resolver.

Reply:
Ok, but why only for` GithubFactoryParametersResolver` ?

Reply:
Ah, I forgot to apply the interface for other resolvers, fixed.